cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=55)
+    description = serializers.CharField(max_length=255)
```

Comment:
The `description` field in the serializer has a `max_length` of 255, but the `Movie` model uses a `TextField`, which does not have a `max_length` restriction. Consider removing the `max_length` constraint from the serializer or adjusting it to reflect any intended limits.